src/utils/validators/graphqlQueryValidators.ts

```
@@ -100,26 +104,78 @@ const swapTransactionValidator = Joi.object({
 export const createDonationQueryValidator = Joi.object({
   amount: Joi.number()?.greater(0).required(),
   transactionId: Joi.when('safeTransactionId', {
-    is: Joi.any().empty(),
-    then: Joi.alternatives().try(
-      Joi.string().required().pattern(txHashRegex, 'EVM transaction IDs'),
-      Joi.string().required().pattern(solanaTxRegex, 'Solana Transaction ID'),
-    ),
-    otherwise: Joi.string()
+    is: Joi.exist(),
+    then: Joi.string()
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)